.github/assign_author.yml

```diff
@@ -0,0 +1 @@
+addAssignees: author
```

Comment:
Can you please explain what this file is doing?

Reply:
I don't see this file growing in the future. Is that correct? If it is, can we remove this file and merge contents with `reviewers.yaml` and perhaps rename it to `auto-assignees.yaml` instead of the previous `auto_assign.yaml`. WDYT?

Reply:
Yes, I think this should work. I agree, it's better.